src/jvm.jl

```diff
@@ -16,17 +16,47 @@ const JNI_ENOMEM       = convert(Cint, -4)              #/* not enough memory */
 const JNI_EEXIST       = convert(Cint, -5)              #/* VM already created */
 const JNI_EINVAL       = convert(Cint, -6)              #/* invalid arguments */
 
+function javahome_winreg()
+    processor_architecture = ENV["PROCESSOR_ARCHITECTURE"]
+    keypath = utf16("")
+    javahome = ""
+    cv = "CurrentVersion"
+    jh = "JavaHome"
+    global const msvclib = "msvcr100.dll"
+    
+    if processor_architecture == "AMD64" 
```

Comment:
I thought that we found at today that the env `PROCESSOR_ARCHITECTURE` is dependant on the word size of the application?